development.md

```diff
@@ -39,6 +39,8 @@ You can use our nightly builds to test new features of upcoming versions. They a
 
 Nightly builds can cause all kinds of crashes and malfunction. Please report such bugs, but don't expect us to deliver a fix right afterwards. If you need stable functionality, use the normal build.
 
+The "light" version is without MLKit translator. It is therefore smaller and without a non [FOSS](https://wikipedia.org/wiki/Free_and_open-source_software) part.
```

Comment:
It still contains other non-FOSS modules (Google Maps, some API, see the (meanwhile closed) issue in c:geo repo about F-Droid version), thus I propose to write "with less" instead of "without".